Cargo.toml

```diff
@@ -5,7 +5,7 @@ resolver = "2"
 members = [
   "bench_util",
   "cli",
-  "cli/napi_sym",
+  "cli/napi/sym",
```

Comment:
Buries napi_sym in the napi folder.

Reply:
Yeah, should be ok. I can do the release this week 😅

Reply:
Sounds good :)